packages/react-components/react-timepicker-compat-preview/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-timepicker-compat-preview",
-  "version": "0.1.9",
+  "version": "0.1.10",
```

Comment:
this shouldn't be done manually , its handled by release pipeline

Reply:
Instead of bumping package.json to create a changefile just add `@deprecated` annotations on all the public API elements being exported - that will force you to generate a changefile anyway.

Make sure to run `npm deprecate` on the package and probably should also be deleted from the repo once the deprecated version is published

Reply:
I updated the compat-preview package to remove this version bump, and adding @deprecate annotation on all public api. 
But I don't think I have the right to run `npm deprecate`. 
I have plan to delete compat-preview from our repo in the next PR.